website/source/docs/templates/engine.html.md

```diff
@@ -22,9 +22,11 @@ need greater than one second granularity, you should use `{{uuid}}`, for
 example when you have multiple builders in the same template.
 
 In addition to globally available functions like `{{timestamp}}`, some
-configurations have special local variables that are available only for
-that configuration. These are recognizable because they're prefixed by a
-period, such as `{{.Name}}`.
+configurations have special template variables that are available only
```

Comment:
Alrighty. :) How about a definition and a link to one of the builder-exposed variables and I'll rewrite the latter example - which is kinda weak anyways?